### ISSUE TITLE
F-Droid inclusion

### ISSUE THREAD

**Issue by IzzySoft** [GittyMac/WalletCount#3]
You've opened [an RFP](https://gitlab.com/fdroid/rfp/-/issues/1336) with us to have your app listed at F-Droid – thanks for that! Unfortunately the app reproducibly crashes whenever one wants to spend money:

```
E AndroidRuntime: FATAL EXCEPTION: main
E AndroidRuntime: Process: com.lako.walletcount, PID: 6959
E AndroidRuntime: java.lang.NumberFormatException: For input string: "100,00"
E AndroidRuntime:    at sun.misc.FloatingDecimal.readJavaFormatString(FloatingDecimal.java:2043)
E AndroidRuntime:    at sun.misc.FloatingDecimal.parseDouble(FloatingDecimal.java:110)
E AndroidRuntime:    at java.lang.Double.parseDouble(Double.java:539)
E AndroidRuntime:    at com.lako.walletcount.SpendFundsSheet$1.onClick(SpendFundsSheet.java:47)
E AndroidRuntime:    at android.view.View.performClick(View.java:6311)
E AndroidRuntime:    at com.google.android.material.button.MaterialButton.performClick(MaterialButton.java:992)
E AndroidRuntime:    at android.view.View$PerformClick.run(View.java:24823)
E AndroidRuntime:    at android.os.Handler.handleCallback(Handler.java:790)
E AndroidRuntime:    at android.os.Handler.dispatchMessage(Handler.java:99)
E AndroidRuntime:    at android.os.Looper.loop(Looper.java:164)
E AndroidRuntime:    at android.app.ActivityThread.main(ActivityThread.java:6719)
E AndroidRuntime:    at java.lang.reflect.Method.invoke(Native Method)
E AndroidRuntime:    at com.android.internal.os.RuntimeInit$MethodAndArgsCaller.run(RuntimeInit.java:449)
E AndroidRuntime:    at com.android.internal.os.ZygoteInit.main(ZygoteInit.java:807)
W ActivityManager:   Force finishing activity com.lako.walletcount/.MainActivity
```

This currently holds back it's listing. It meets all criteria and builds fine – but… well, if it can't be used we cannot list it. I have a vague idea this has to do with locales, and you didn't expect a non-US setting there: I never put in any comma (the app wouldn't even let me IIRC) – and "100,00" would be the typical DE way while US would have it "100.00".

Any chance you get this fixed, so we can get that RFP processed finally? Thanks!

**Comment by IzzySoft** [comment]
@GittyMac any word?

**Comment by GittyMac** [comment]
I apologize for the late response.  I will look into the issue and hopefully fix it this week.

**Comment by IzzySoft** [comment]
Thanks a lot, and :crossed_fingers:

**Comment by GittyMac** [comment]
I published v1.1, which should fix the issue.

**Comment by IzzySoft** [comment]
Thanks! Confirmed: no more crashes. Will finalize the MR at F-Droid then. One remaining issue is with the budget view:

[<img src="https://user-images.githubusercontent.com/6781438/106357631-8b6c8d00-6307-11eb-8868-f7d7c96b1581.png" width="200" alt="walletcount">](https://user-images.githubusercontent.com/6781438/106357631-8b6c8d00-6307-11eb-8868-f7d7c96b1581.png)

Note the screen is not scrollable, so it's not readible what stands there – and if there are any action buttons, I cannot see them. How I got there: added 50 to the budget, spent 15. Wonder why now there's a negative budget – but maybe misunderstood the app's principles. Screenshot has original resolution of the test device, in case it matters (480x854). No landscape support it seems – but that might be intended.
